distributed/client.py

```diff
@@ -2979,7 +2981,7 @@ def run(
     @staticmethod
     def _get_computation_code(
         stacklevel: int | None = None, nframes: int = 1
-    ) -> tuple[str, ...]:
+    ) -> tuple[SourceCode, ...]:
```

Comment:
Would be good to update the corresponding type annotation on the scheduler side

Reply:
Sure, I can do that. Assume we're talking about the destination of `_send_to_scheduler` schema definition somewhere? 
But maybe this depends on a later PR if we want to propagate out the line number from your other comment? Here I was only intending to capture it for @shughes-uk if she has something specific in mind. Presently, just propagating it out would terminate somewhere with just passing the code text b/c nothing is 'hooked up' to make use of the line number.

Reply:
Updated. :+1: